src/library/Hooks/usePoolCommission/index.tsx

```diff
@@ -0,0 +1,24 @@
+// Copyright 2023 @paritytech/polkadot-staking-dashboard authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import { useBondedPools } from 'contexts/Pools/BondedPools';
+import { usePoolsConfig } from 'contexts/Pools/PoolsConfig';
+
+export const usePoolCommission = (id: number) => {
+  const { getBondedPool } = useBondedPools();
+  const bondedPool = getBondedPool(id);
+  const { stats } = usePoolsConfig();
+  const { globalMaxCommission } = stats;
+
+  const getCommission = () => {
+    const currentCommission = Math.min(
+      Number(bondedPool?.commission?.current?.[0].slice(0, -1)),
```

Comment:
This could still return `undefined`. Finish by saying `|| 0`.
```suggestion
      Number(bondedPool?.commission?.current?.[0]?.slice(0, -1) || 0),
```

Reply:
I have added the change, but I think the "if (currentCommission)" here can filter the undefined so we don't need to add this, no?